books/RayTracingTheRestOfYourLife.html

```diff
@@ -99,14 +128,14 @@
                 inside_circle++;
         }
         std::cout << std::fixed << std::setprecision(12);
-        std::cout << "Estimate of Pi = " << 4*double(inside_circle) / N << '\n';
+        std::cout << "Estimate of Pi = " << 4 * double(inside_circle) / N << '\n';
```

Comment:
To simplify the code, what do you think about `(4.0 * inside_circle) / N`?

Reply:
unambiguously better.